### PR TITLE
ENH - Improve docs structure and update content

### LINKED ISSUE #47: Contribution guide
Create a contribution guide to help people who want to add knowledge to the repo. 

I think it can be usefull for storing things in the best way or help the team to get all informations at the same point.

### LINKED ISSUE #105: Link to the public meeting notes looks outdated
The current public notes point to an issue in the `jupyterlab/team-compass` that has been closed:

https://jupyter-accessibility.readthedocs.io/en/restructure/README.html#join-our-jupyterlab-accessibility-meetings

image

Currently pointing to https://github.com/jupyterlab/team-compass/issues/98

### PR DESCRIPTION
**Pull request by trallard**
## Changes introduced:

1. Simplify the documentation building requirements
2. Document how to build the docs
3. Restructure docs content - better sections and headings
4. Update requirements
5. Update logo
6. Ensure meeting notes are rendered

## Outstanding

- [x] There are still some items missing - marked as TODO in the corresponding files
- [x] Enable Read the docs previews as per https://docs.readthedocs.io/en/stable/pull-requests.html - blocked on access to RTC cc/ @choldgraf  and @blink1073 
---

Closes #105 
Closes #47 
It partially tackles #86

### CODE REVIEW

File: README.md
Comment (choldgraf) [inline README.md:15]: minor non-blocking suggestion - I've found it easiest to use [semantic line breaks](https://sembr.org/) to make diffs for docs PRs easier to manage and read. My rule is generally "one line per sentence".

File: README.md
Comment (choldgraf) [inline README.md:21]: should we just not recommend it then?

File: README.md
Comment (choldgraf) [inline README.md:24]: Random Q: aren't emojis before the header text non-a11y friendly? I thought I remember reading somewhere that these are nuisances for people who have to read the header text from start-to-finish via audio

**Review by choldgraf — COMMENTED** [review]
A few quick comments and questions in there but nothing major. I've also added @trallard as maintainer on the RTD repo, and enabled PR builds so the next commit push should trigger a PR preview.

I took a quick look at the changes and they all seem good to me, in my opinion since this is a refactor, we should take a look at the PR build preview and make sure it looks reasonable, and if so then we should just merge it and iterate from there!

File: README.md
Comment (trallard) [reply README.md:15]: Agree - 👍🏽  will fix this once done with the rest of the bulk

File: README.md
Comment (trallard) [reply README.md:21]: 🤷🏻‍♀️ I am tempted to say no - but AFAIK this is still the only extension of its type. I added the warning as others might feel differently about removing this.

cc @isabela-pf

File: README.md
Comment (trallard) [reply README.md:24]: I cannot remember if this is correct - maybe @isabela-pf  knows more about - in which case we can remove them or move them to the end

**Comment by choldgraf** [comment]
Also a couple quick thoughts on this PR:

- Looking at the structure of the site pages, I'm not sure who this documentation is for. Is it for others in the Jupyter ecosystem to learn about accessibility? Is it a way for us to provide accountability for the EOSS4 a11y project?
- Right now, the most prominent links to the left are about EOSS4...I feel like these are probably not the most relevant ecosystem-wide links to display, compared with accessibility resources links etc (though this depends on the answer to the above question).
- The "Join the Jupyter accessibility community" page is more like a list of previous accessibility workshops and resources, rather than documentation about how to join. Maybe we can rename `contribute` to `Contribute to Jupyter Accessibility`, move it to the top of the section, and rename the current `Contribute to...` sections above under the `Accessibility efforts` section?
- The EOSS4 roadmap could probably be two nested pages under a top-level page that describes the EOSS4 project in general, rather than two top-level pages.

**Comment by trallard** [comment]
> Also a couple quick thoughts on this PR:
> 
> * Looking at the structure of the site pages, I'm not sure who this documentation is for. Is it for others in the Jupyter ecosystem to learn about accessibility? Is it a way for us to provide accountability for the EOSS4 a11y project?

So these docs now cover a few scenarios:

1. Go-to place for folks within Jupyter wanting to learn more about accessibility - need to add more resources here as the one suggested in #15 
2. Single point to learn about the "Jupyter-wide" accessibility initiatives and ongoing work - as we see it, the EOSS a11y project is for and by the Jupyter community (even if folks undertaking the work are in Quansight) - we decided from day 1 that we would work in the open and keep folks aligned on what is happening, changes in plans and treat the grant work as I mentioned before: for and by the community

> * Right now, the most prominent links to the left are about EOSS4...I feel these are probably not the most relevant ecosystem-wide links to display, compared with accessibility resources links etc. (though this depends on the answer to the above question).

I see - this can be misleading - I will move this to a lower level, maybe under "funded projects" or something of the such. 

> * The "Join the Jupyter accessibility community" page is more like a list of previous accessibility workshops and resources rather than documentation about how to join. Maybe we can rename `contribute` to `Contribute to Jupyter Accessibility`, move it to the top of the section, and rename the current `Contribute to...` sections above under the `Accessibility efforts` section?

Good idea tbh this repo still needs a good reorg - right now my goal was to get the meeting minutes into the docs as everything was broken and Isabela had tried multiple times to add them to the TOC. And get those builds working again. 
The structure still needs plenty of work and some internal sections' content - but that can be done now in more minor PR's.
 

> * The EOSS4 roadmap could probably be two nested pages under a top-level page that describes the EOSS4 project in general, rather than two top-level pages.
Agreed - this roadmap also needs updating (cc @isabela-pf  as this has been something you've kept up-to-date in the past)

**Comment by jasongrout** [comment]
Chris: done!

**Review by choldgraf — APPROVED** [review]
Sounds good @trallard - this PR looks good to me, there are more things to improve but IMO don't need to block this one.

I'll await @isabela-pf 's approval before merging though, as she has more visibility on the accessibility efforts and strategy than I do!

File: README.md
Comment (isabela-pf) [reply README.md:21]: I think it may be worth keeping in case people are working with something compatible with this tool. In my experience, lots of people are not using the most recent Jupyters and this could be a big help.

File: README.md
Comment (isabela-pf) [reply README.md:24]: I was able to find some sources to support this (there's a mention of the way it will be read in a [Perkins School for the blind blog post](https://www.perkins.org/resource/how-do-people-vision-impairments-use-emoji/)[Easter Seals blog post](https://blog.easterseals.com/emojis-and-accessibility-the-dos-and-donts-of-including-emojis-in-texts-and-emails/), [UX Collective blog post](https://uxdesign.cc/emojis-in-accessibility-how-to-use-them-properly-66b73986b803)). It seems that it's generally considered more friendly to have the words first in the case of "important" (in this case navigational) content. 

Do we want to remove the emojis or add them to the end of the header? I'm happy to make the change with whatever decision.

File: docs/community/index.md
Comment (isabela-pf) [inline docs/community/index.md:1]:
```suggestion
# Calls and Events
```

It may be my bias, but I think this phrasing could be confused with the more broad [Jupyter Community Calls](https://discourse.jupyter.org/t/jupyter-community-calls/668/) since I already have people asking me about other calls that use this language.

**Review by isabela-pf — APPROVED** [review]
There are a lot of changes here, so I can't go through with a fine toothed comb at the moment. They all seem fine to me, and I agree this can be merged as is.

I think [@choldgraf's latest review comment](https://github.com/jupyter/accessibility/pull/106#issuecomment-1211947525) also makes sense, especially noting the prominence of the CZI EOSS4 grant. I don't think that's the key thing in this repo, and it definitely won't be when the grant is completed. I'm happy to see these changes here, or I'll copy them over to an issue if they don't get filled in this PR. Thank you for bringing those up.

I'm happy to do the merge after re/moving the the emojis in the headers. Do we know which is preferred?

**Comment by trallard** [comment]
I have made the reorg based on @choldgraf so now the grant work is under the "funded work" section. 

I will get rid of the emojis tomorrow and merge then  🚀

**Pull request merged**
